geode-junit/src/main/java/org/apache/geode/internal/cache/DiskRegionTestingBase.java

```diff
@@ -200,6 +202,19 @@ protected void deleteFiles() {
     tempDir.delete();
   }
 
+  protected void forceDeleteFiles() {
+    closeDiskStores();
+    File file = tempDir.getRoot();
+    File[] files = file.listFiles();
+    for (File each : files) {
+      try {
+        FileUtils.forceDelete(each);
+      } catch (IOException e) {
+        LogService.getLogger().error("Caught", e);
```

Comment:
Instead of logging the error, can we add and use ErrorCollector? I would just add this to DiskRegionTestingBase up where the other Rules are defined.
```
  @Rule
  public ErrorCollector errorCollector = new ErrorCollector();
```
And then:
```
      try {
        FileUtils.forceDelete(each);
      } catch (IOException e) {
        errorCollector.addError(e);
      }
```
After each test finishes, (if any were thrown) ErrorCollector will fail the test and print out every stack trace.

Since SimpleDiskRegionJUnitTest is an integration test, logging the stack traces won't cause the test to fail like a distributed test would.

Reply:
I'd definitely like to do something more than just logging the error, but not all of the files are deleted (like `statsstats.gfs` file).  The `statsstats.gfs` file doesn't have to be deleted, though.  We wrapped the `FileUtils.forceDelete(each);` method line with a `if (!each.getName().contains(".gfs"))` since that file doesn't have to be deleted.  Let me know what you think!